drivers/adc/ad719x/ad719x.h

```diff
@@ -127,8 +127,8 @@
 #define AD719X_CH_TEMP   8
 #define AD719X_CH_SHORT  9
 
-/* ID Register Bit Designations (AD7193_REG_ID) */
-#define AD719X_ID_MASK          0x0F
```

Comment:
for AD7193, for example, there are only 4 bits for the chip id. the rest of them are undetermined:
![image](https://user-images.githubusercontent.com/26061529/153402937-8cdf1659-6192-4810-8a9d-50e4bb489a12.png)

This means that the 4 MSB can be either 0 or 1, depending on the chip default configuration.
That might cause some issues at initialization if you expand the mask for all the chips supported by this driver.
I'd rather change the mask inline, depending on the part name.

Reply:
Or you can leave it as it is (`0x0F`), since for ad7195 the 4LSB are different from any other supported part.

Reply:
> Or you can leave it as it is (`0x0F`), since for ad7195 the 4LSB are different from any other supported part.

What if, in the near future, another ad719x chip will be released with ID 0x06?

Fixed ID mask inline.

Reply:
> > Or you can leave it as it is (`0x0F`), since for ad7195 the 4LSB are different from any other supported part.
> 
> What if, in the near future, another ad719x chip will be released with ID 0x06?
> 
> Fixed ID mask inline.

I agree, separate masks is a better solution